common/djangoapps/third_party_auth/migrations/0001_initial.py

```diff
@@ -59,18 +58,6 @@ class Migration(migrations.Migration):
                 'verbose_name_plural': 'Provider Configuration (OAuth)',
             },
         ),
-        migrations.CreateModel(
```

Comment:
As noted in Slack, I can't think of a better way to do this than what you did, but it would be great to just get @feanil, or one more opinion, on this one to make sure.  Thanks.

Reply:
I talked this over with @feanil , and we're planning on including this change here, then squashing all migrations in this djangapp once the removal migration goes in: https://openedx.atlassian.net/browse/BOM-1355

Reply:
👍

Reply:
:+1:

Reply:
This is causing issues with following migration https://github.com/edx/edx-platform/blob/57f7f50b09a0fc5743eb6346bbfb0a380152a739/common/djangoapps/third_party_auth/migrations/0025_auto_20200302_1701.py on bokchoy test envs and on local devstack.